src/user/admin.js

```diff
@@ -70,7 +70,12 @@ module.exports = function (User) {
 
 			let line = '';
 			usersData.forEach((user, index) => {
-				line += `${fields.map(field => user[field]).join(',')}`;
+				line += `${fields.map(field => {
+					if ( /^\d+$/.test(user[field])) {
+						return `'${user[field]}'`;
+					}
+					return user[field]
+				}).join(',')}`;
```

Comment:
I'd replace this with `isFinite(user[field]) ? `'${user[field]}'` : user[field]`

Using a regular expression is a bit more expensive.